static/scss/variables.scss

```diff
@@ -2,6 +2,9 @@ $static-path: '/static';
 
 $primary: #a31f34;
 $gray: #363636;
+$red: #a31f34;
+$light-gray: #f5f5f5;
```

Comment:
Looks like this maybe got defined in a different PR on line 10, which is overriding your value defined here.